core/tests/protractor.conf.js

```diff
@@ -338,6 +339,26 @@ exports.config = {
       }));
     }
 
+    // Adding a video reporter. For more information see
+    // https://www.npmjs.com/package/protractor-video-reporter
+
+    var ffmpegArguments = [
+      '-y',
+      '-r', '30',
+      '-f', 'x11grab',
+      '-s', '1366x768',
+      '-i', process.env.DISPLAY,
+      '-g', '300',
+      '-vcodec', 'qtrle',
+    ]
+
+    jasmine.getEnv().addReporter(new VideoReporter({
+      baseDirectory: path.resolve(__dirname, '../../../protractor-video'),
+      singleVideoPath: 'uuid',
+      saveSuccessVideos: true,
```

Comment:
If we're going to use UUIDs to name the videos, we might want to not save success videos. That will reduce the number of videos we have to go through to find the one we want